update faq.md (#3336)
juicedata/juicefs: docs/en/faq.md

Different types of JuiceFS clients have different ways to obtain logs. For details, please refer to ["Client log"](administration/fault_diagnosis_and_analysis.md#client-log) document.

### Can JuiceFS directly read files that already exist in object storage?

JuiceFS cannot directly read files that already exist in object storage. Although JuiceFS typically uses object storage as the data storage layer, it is not a tool for accessing object storage in the traditional sense. You can refer to the [technical architecture](introduction/architecture.md) documentation for more details.

> If you want to migrate existing data in an object storage bucket to JuiceFS, you can use [JuiceFS Sync](guide/sync.md).

Author: ```suggestion
If you want to migrate existing data in an object storage bucket to JuiceFS, you can use [`JuiceFS Sync`](guide/sync.md).
```

> ### I have four servers and I would like to know if it is possible to utilize JuiceFS to merge them into a singular, independent file system for practical use?

Author: ```suggestion
### How can I combine multiple servers into a single JuiceFS file system for use?
```
